### PR TITLE
Added gcloud yaml and comment features to bot

### PR DESCRIPTION
**Pull request by hybridx**
Chore/add gcloud

### CODE REVIEW

File: app.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+runtime: nodejs
+env: flex
```
Comment (gautamkrishnar) [inline app.yaml:2]: Why do you used a flex instance, cant we use `standard` one.

Comment (gautamkrishnar) [reply app.yaml:2]: https://cloud.google.com/appengine/docs/nodejs

Comment (hybridx) [reply app.yaml:2]: @gautamkrishnar Flex can also be downscaled which is cost effective it also has support for background threads and has a support for docker

Comment (hybridx) [reply app.yaml:2]: @gautamkrishnar Should we still use the `standard` ?

**Comment by gautamkrishnar** [comment]
LGTM

**Review by gautamkrishnar — APPROVED** [review]

**Pull request merged**